server/src/main/java/com/objectcomputing/checkins/security/UserPasswordAuthProvider.java

```diff
@@ -0,0 +1,30 @@
+package com.objectcomputing.checkins.security;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.security.authentication.*;
+import io.reactivex.Flowable;
+import org.reactivestreams.Publisher;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Requires(env = {"local"})
```

Comment:
This means this bean will only exist when MICRONAUT_ENVIRONMENTS is set to local

Reply:
Doesn't this need to be local and test?

Reply:
And ditto for the UserStore?

Reply:
@mkimberlin
Not necessarily, we could split it up that way, but then application-test.yml would be a subset of application-local.yml.

We could just set MICRONAUT_ENVIRONMENTS=local when running tests.

It depends on design questions like are we going to have tests that use OAuth to make sure it is working as expected?

Let me know the thoughts on what testing in the future we may want and I can make this a little more future proof/configurable. 

I can see pros and cons either way, but at the very least I agree "local" would be a lie if we were testing in GitHub. =)

Reply:
I'll just do as you said for now, it makes sense enough. I think we may want to come back at this later in another story when we increase our testing footprint.

Reply:
@mkimberlin Updated